addons/mail_group/models/mail_group.py

```diff
@@ -251,6 +251,11 @@ def _alias_get_error_message(self, message, message_dict, alias):
         self.ensure_one()
 
         if alias.alias_contact == 'followers':
+            # Selected group of users
+            email = email_normalize(message_dict.get('email_from'))
+            if self.access_mode == 'groups' and \
+               bool(self.search([('id', '=', self.id), ('access_group_id.users.email_normalized', '=', email)], limit=1)):
+                return  # Skip the verification because the partner is in the user group list
```

Comment:
Would still like to fail here instead of relying on the next check to give an error message.

Maybe something of this order?

```suggestion
            if self.access_mode == 'groups' and \
               self.search([('id', '=', self.id), ('access_group_id.users.email_normalized', '=', email)], limit=1):
                return  # Skip the verification because the partner is in the user group list
            elif self.access_model == 'members' and self._find_member(message_dict.get('email_from', '')):
                return
             else:
                 return _('Only members can send email to the mailing list.')
```

Reply:
I think we can even remove the check for `if alias.alias_contact == 'followers':` no ? (`access_model` is enough / better)